crates/rust-eigenda-v2-client/src/lib.rs

```diff
@@ -84,9 +89,11 @@ mod tests {
         0x9e, 0x74, 0x98, 0x92, 0x54,
     ]);
     pub const CERT_VERIFIER_ADDRESS: H160 = H160([
-        0xfe, 0x52, 0xfe, 0x19, 0x40, 0x85, 0x8d, 0xcb, 0x6e, 0x12, 0x15, 0x3e, 0x21, 0x04, 0xad,
-        0x0f, 0xdf, 0xbe, 0x11, 0x62,
+        0xd3, 0x05, 0xae, 0xbc, 0xde, 0xc2, 0x1d, 0x00, 0xfd, 0xf8, 0x79, 0x6c, 0xe3, 0x7d, 0x0e,
+        0x74, 0x83, 0x6a, 0x6b, 0x6e,
```

Comment:
isn't there a more easily maintainable way to load this? Like a macro to do hex string -> H160 transform perhaps?

Reply:
or can we just use a &str like we do the below addresses? Why are some of them &strs and some of the H160?